src/vivarium_gates_iv_iron/data/loader.py

```diff
@@ -135,6 +157,126 @@ def _load_em_from_meid(location, meid, measure):
 
 
 # TODO - add project-specific data functions here
+def load_pregnancy_incidence_rate(key: str, location: str):
+
+    not_pregnant = get_prevalence_not_pregnant(key, location)
```

Comment:
It's better to call `get_data` to get these values in case there is a change in how these values are calculated (the researchers might change their minds about how to calculate ectopic pregnancy rate so that you can no longer use `load_standard_data`. If you call `get_data`, you only need to change that in one place, rather than remembering everything that depends on it.